moto/events/models.py

```diff
@@ -1437,26 +1474,86 @@ def delete_connection(self, name):
         return connection.describe_short()
 
     def create_api_destination(
-        self, name, description, connection_arn, invocation_endpoint, http_method
+            self, name, description, connection_arn, invocation_endpoint, invocation_rate_limit_per_second, http_method
     ):
+        """
+        Creates an API destination, which is an HTTP invocation endpoint configured as a target for events.
+        Docs:
+            https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_CreateApiDestination.html
 
+        Returns:
+            dict
+        """
         destination = Destination(
             name=name,
             region_name=self.region_name,
             description=description,
             connection_arn=connection_arn,
             invocation_endpoint=invocation_endpoint,
+            invocation_rate_limit_per_second=invocation_rate_limit_per_second,
             http_method=http_method,
         )
 
         self.destinations[name] = destination
-        return destination
+        return destination.describe_short()
 
     def list_api_destinations(self):
         return self.destinations.values()
 
     def describe_api_destination(self, name):
-        return self.destinations.get(name)
+        """
+        Retrieves details about an API destination.
+        Docs:
+            https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_DescribeApiDestination.html
+        Args:
+            name: The name of the API destination to retrieve.
+
+        Returns:
+            dict
+        """
+        destination = self.destinations.get(name)
+        if not destination:
+            raise ResourceNotFoundException("An api-destination '{}' does not exist.".format(name))
+        return destination.describe()
+
+    def update_api_destination(self, *, name, **kwargs):
+        """
+        Creates an API destination, which is an HTTP invocation endpoint configured as a target for events.
+        Docs:
+            https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_UpdateApiDestination.html
+
+        Returns:
+            dict
+        """
+        destination = self.destinations.get(name)
+        if not destination:
+            raise ResourceNotFoundException("An api-destination '{}' does not exist.".format(name))
+
+        for attr, value in kwargs.items():
+            if value is not None and hasattr(destination, attr):
+                setattr(destination, attr, value)
+        return destination.describe_short()
+
+    def delete_api_destination(self, name):
+        """
```

Comment:
nitpick: I don't think we necessarily need to add the links to the API docs for each new method we're adding. It doesn't harm to have them, but I think it's simply not required. (Remember - this entire project is about providing AWS API methods, so most people are probably aware where to look up the API specs ;) )

Reply:
> this entire project is about providing AWS API methods
:joy: Fair! Thanks for the feedback! Will not include in future PRs :grimacing: